README.md

```diff
@@ -323,43 +396,92 @@ An Oracle model definition consists of the following properties:
           "nullable":"N"
         }
       },
-      "available":{
-        "type":"Number",
-        "required":false,
-        "length":22,
+    }
+  }
+```
+
+</details>
+
+**Notice**: the Oracle database default type is UPPERCASE. If the oracle settings are not specified in the model, for example:
+
+```ts
+export class Demo extends Entity {
+  @property({
+    type: 'number',
+    required: true,
+    id: true,
+  })
+  id: number;
+```
+
+the connector would look for a table named `DEMO` under the default schema in the database and also tend to map the id property to a column named `ID` in that table. This might cause errors if the default table/colum name doesn't exist. Please do specify the settings if needed.
```

Comment:
maybe omit `tend to`? 

```suggestion
the connector would look for a table named `DEMO` under the default schema in the database and also map the id property to a column named `ID` in that table. This might cause errors if the default table/colum name doesn't exist. Please do specify the settings if needed.
```